packages/cli/test/unit/commands/projects.test.ts

```diff
@@ -0,0 +1,266 @@
+import { join } from 'path';
+import fs from 'fs-extra';
+import projects from '../../../src/commands/projects';
+import { useUser } from '../../mocks/user';
+import { useTeams } from '../../mocks/team';
+import { defaultProject, useProject } from '../../mocks/project';
+import { client } from '../../mocks/client';
+import { Project } from '../../../src/types';
+
+describe('projects', () => {
+  describe('connect', () => {
+    const originalCwd = process.cwd();
+    const fixture = (name: string) =>
+      join(__dirname, '../../fixtures/unit/commands/projects/connect', name);
+
+    it('connects an unlinked project', async () => {
+      const cwd = fixture('unlinked');
+      try {
+        process.chdir(cwd);
+        await fs.rename(join(cwd, 'git'), join(cwd, '.git'));
+        useUser();
+        useTeams('team_dummy');
+        useProject({
+          ...defaultProject,
+          id: 'unlink',
+          name: 'unlink',
+        });
+        client.setArgv('projects', 'connect', '--cwd', cwd);
```

Comment:
You're already doing `process.chdir()` so these `--cwd` flags shouldn't be necessary (on the tests below as well):

```suggestion
        client.setArgv('projects', 'connect');
```